pages/common/ipython.md

```diff
@@ -1,7 +1,7 @@
 # IPython
 
 > A Python shell with automatic history, dynamic object introspection, easier configuration, command completion, access to the system shell and more.
-> More information: <https://ipython.org/documentation.html>.
+> More information: <https://ipython.readthedocs.io/en/stable>.
```

Comment:
Maybe we should use the home and it redirects to the proper language :thinking: 
```suggestion
> More information: <https://ipython.readthedocs.io>.
```

Reply:
Good point